Vizzuality/heco-invest
# feat: allow to switch between Jobs and API/Backoffice routes

I am adding ability to enable/disable appropriate set of routes based on ENV variables. Current routes groups are:
 - IS_API_INSTANCE --> enables API and Backoffice routes
 - IS_JOBS_INSTANCE --> enables JOBS routes

Please note that route `post :test, to: ->(_env) { [200, {}, [""]] }` is here just for testing purposes, because I wanted to be able to test that everything works. This one will be removed after we have some logic at `jobs` controllers.

## Testing instructions

There is already test for this, but basically set `IS_JOBS_INSTANCE = true` and check that `/jobs/test` returns `success`

## Tracking

https://vizzuality.atlassian.net/browse/LET-259
